docs/changes/io.fits/17552.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix ```Table.write(..., overwrite=True)``` when using a pathlib filename
```

Comment:
```suggestion
Fix ``Table.write(..., format="fits", overwrite=True)`` when filename is provided as ``pathlib.Path``.
```

Reply:
This thread was marked as resolved but it looks like these suggestions were not yet taken into account. I'm unresolving it.

Reply:
Pretty sure I saw the accepted changes in a diff at one point. Was it lost after force push? 🤔

Reply:
Yes... my bad ... I rebased without pulling... 🙄